static/css/restyle.less

```diff
@@ -711,24 +717,24 @@ button.good {
     background: 0;
 
     .hovercard.addon {
-         &:hover,
-         &:hover:before,
-         &:hover .icon,
-         &:hover .summary,
-         &:hover .more {
-            background: none;
-            border-color: transparent;
-            box-shadow: none;
-         }
-
-         .more {
-             display: block !important;
-             .addon-summary,
-             .byline,
-             .vitals {
-                display: none;
-             }
+      &:hover,
+      &:hover:before,
+      &:hover .icon,
+      &:hover .summary,
+      &:hover .more {
+        background: none;
+        border-color: transparent;
+        box-shadow: none;
+      }
+
+      .more {
+         display: block !important;
```

Comment:
I usually put a space before further nesting. Helps to separate out the rules from additional nested selectors.